detection_operator_runner.py

```diff
@@ -24,15 +27,33 @@
 import pylot.utils
 import pylot.simulation.utils
 from pylot.perception.camera_frame import CameraFrame
-from pylot.drivers.sensor_setup import RGBCameraSetup
+from pylot.perception.depth_frame import DepthFrame
+from pylot.perception.segmentation.segmented_frame import SegmentedFrame
+from pylot.drivers.sensor_setup import RGBCameraSetup, DepthCameraSetup, SegmentedCameraSetup
 
 _lock = threading.Lock()
 
 FLAGS = flags.FLAGS
+flags.DEFINE_string('test_operator',
+                    'detection_operator',
+                    help='Operator of choice to test')
```

Comment:
Could you make this an enum, similar to here?
https://github.com/erdos-project/pylot/blob/765bbfa59812006961f5868bcb0a240cc6770b98/pylot/flags.py#L43-L46